docs/rfcs/0001/readme.md

```diff
@@ -0,0 +1,65 @@
+Rabbit Request For Comment Process
+==================================
+
+Rabbit software must be designed in close collaboration our end-users. Part of this process involves open discussion in the form of Request For Comment (RFC) documents. The remainder of this document presents the RFC process for Rabbit.
+
+History & Philosophy
+-------------------
+
+NNF RFC documents are modeled after the long history of IETF RFC documents that describe the internet. The philosophy is captured best in [RFC 3](https://www.rfc-editor.org/rfc/rfc3)
+
+> The content of a NWG note may be any thought, suggestion, etc. related to
```

Comment:
We should define NWG

Reply:
Distracting bits like that should be elided.   "The content of a [...] note may be any ..."